src/SDKs/Search/DataPlane/Microsoft.Azure.Search/Microsoft.Azure.Search.csproj

```diff
@@ -5,71 +5,19 @@
     <Description>Makes it easy to develop a .NET application that uses Azure Search.</Description>
     <AssemblyTitle>Microsoft Azure Search Library</AssemblyTitle>
     <AssemblyName>Microsoft.Azure.Search</AssemblyName>
-    <VersionPrefix>3.0.3</VersionPrefix>
-    <PackageTags>Microsoft Azure Search;Search</PackageTags>    
+    <VersionPrefix>3.0.4</VersionPrefix>
+    <PackageProjectUrl>https://github.com/Azure/azure-sdk-for-net</PackageProjectUrl>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/Microsoft/dotnet/master/LICENSE</PackageLicenseUrl>
+    <PackageIconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</PackageIconUrl>
+    <PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
```

Comment:
@brjohnstmsft you do not need these.
Not having these here allows us to change behavior of common nuget properties, especially license url, project url and icon.
These properties are implicitly added to every nuget package.